plugin/evm/vm.go

```diff
@@ -355,6 +359,10 @@ func (vm *VM) Initialize(
 
 	log.Info("Initializing Coreth VM", "Version", Version, "Config", vm.config)
 
+	if deprecateMsg != "" {
```

Comment:
Can we put the declaration of `deprecateMsg` inside of the if statement so that it's all in one place?

Reply:
see the comment:
```
// We should deprecate config flags as the first thing, before we do anything else
// because this can set old flags to new flags. log the message after we have
// initialized the logger.
```